src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  flex-direction: column;
+  display: flex;
+  justify-content: center; /* Вирівнює по горизонталі */
+  align-items: center; /* Вирівнює по вертикалі */
+  height: 100vh;
   margin: 0;
 }
+
+@import 'main';
```

Comment:
The `@import` rule is used to include the `main` SCSS file. Make sure that the `main.scss` file exists in the same directory or provide the correct path. Note that `@import` is deprecated in favor of `@use` and `@forward` in newer versions of Sass, which offer better modularity and avoid potential conflicts.